Makefile

```diff
@@ -1,5 +1,5 @@
 build:
-	mvn test-compile -DskipTests=true -B -V
+	mvn spotless:check test-compile -DskipTests=true -B -V
```

Comment:
got annoyed with catching style failures way late in the test process, so running this first to weed out silly stylechecks early in the build